src/App.js

```diff
@@ -1,23 +1,40 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Character from "./components/Character/Character";
+import Display from "./components/Display/Display";
+import Picker from "./components/Picker/Picker";
+import React, { useState } from "react";
 
 function App() {
+  const [head, setHead] = useState("");
+  const [middle, setMiddle] = useState("");
+  const [pants, setPants] = useState("");
+  const [newPhrase, setNewPhrase] = useState("");
+  const [library, setLibrary] = useState([]);
+
+  const handleClick = () => {
+    setLibrary((prevState) => [...prevState, newPhrase]);
+  };
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <main>
+        <header className="App-header">
+          <h1>Frankenstein Laboratory</h1>
```

Comment:
You can update your app.test.js to look for things like this header in the expect statement